meetup/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from pizzaplace.models import PizzaPlace
+from django.core.validators import RegexValidator
+from meetup.services.meetup_api_lookup_agent import MeetupApiLookupAgent
+from django.core.exceptions import ValidationError
+
+
+def validate_urlname(link):
+  validator = RegexValidator(
+    regex='meetup\.com\/\w+(-\w+)*\/$',
+    message="Does not conform to Meetup Url",
+    code='invalid_url')
+  return validator(link)
+
+def validate_meetup_exists(link):
+  looker = MeetupApiLookupAgent(link)
+  is_real = looker.is_real_meetup()
+
+  if not is_real:
+    raise ValidationError("That's not a meetup")
+
+
+
+
+class Meetup(models.Model):
+  name = models.CharField(max_length=500, null=False, blank=False, default=None, unique=True)
```

Comment:
Do you understand the difference between `blank` and `null` on a `CharField`? (See: https://docs.djangoproject.com/en/1.9/ref/models/fields/#blank).

What are the default values of `blank` and `null` for a `CharField`?